src/main/java/org/openstreetmap/atlas/geography/atlas/items/Edge.java

```diff
@@ -37,6 +39,18 @@ protected Edge(final Atlas atlas)
         super(atlas);
     }
 
+    @Override
+    public JsonObject asGeoJsonFeature()
+    {
+        final JsonObject feature = super.asGeoJsonFeature();
```

Comment:
I feel like this should override ```getGeojsonProperties()``` referenced above.

Reply:
LineItem does indeed call that now. That is super. The super makes most of what we need, here, we're adding the start and end nodes to the properties.